### DISCUSSION TITLE
You must select 2 or more models to run ensemble.

### DISCUSSION THREAD

**Discussion by w273268045** [Anjok07/ultimatevocalremovergui#501] [Q&A]
![QQ截图20230417173336](https://user-images.githubusercontent.com/51728674/232445407-f5a5a816-7558-4f2c-9c19-93154e62aaf5.png)
how to treat

**Comment by PURE281** [comment]
easy to solve，click the right frame(available models)，try to choose 2 models (when y click the model,will show the blue )

**Reply by w273268045** [reply]
78
